extra/service_startup.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+set -e
+
+if [[ -e /root/tmp/certbot.sh ]]; then
+    /bin/bash /root/tmp/certbot.sh
```

Comment:
Once we execute this, we can clear the file, so there is no risk on generating a new certificate again.